.changeset/subscribe-start-from.md

```
@@ -0,0 +1,25 @@
+---
+'@mastra/core': minor
+'@mastra/redis-streams': minor
+---
+
+Add `SubscribeOptions.startFrom` so a subscription can begin at the current tail
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)